src/main/java/com/thealgorithms/dynamicprogramming/TravellingSalesMan.java

```diff
@@ -0,0 +1,48 @@
+import java.util.Arrays;
+
+public class TravellingSalesMan {
+
+    public static int tsp(int[][] graph, int mask, int pos, int n, int[][] dp) {
```

Comment:
I think this method could be `private` - what is your opinion?

```suggestion
    private static int tsp(int[][] graph, int mask, int pos, int n, int[][] dp) {
```